src/view.js

```diff
@@ -313,6 +313,15 @@ Marionette.View = Backbone.View.extend({
     if (normalizedChildEvents && _.isFunction(normalizedChildEvents[eventName])) {
       normalizedChildEvents[eventName].apply(layoutView, callArgs);
     }
+
+    // call the parent view's proxyEvent handlers
+    var childTriggers = Marionette.getOption(layoutView, 'childTriggers');
+    childTriggers = Marionette._getValue(childTriggers, layoutView);
```

Comment:
ha yes!  I [totally agree](https://github.com/marionettejs/backbone.marionette/pull/2819#issuecomment-156258456)

I also wonder if we should allow `_getValue` to accept a string like:
```js
Marionette._getValue = function(value, context, params) {
    if(_.isString(value)){
      value = Marionette.getOption(context, value);
    }
    if (_.isFunction(value)) {
      value = params ? value.apply(context, params) : value.call(context);
    }
    return value;
  };
```

We don't currently have a use-case where the we'd want the value to sometimes actually be a string value..  but I suppose we could.. in which case this wouldn't work.